index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>

```

Comment:
- [ ] **[OPTIONAL] Good use of GitHub inline comments 👍** However, It would be better to use a couple of emojis or gifs in the review.

* They make the code review more human-friendly 👽